README.md

```diff
@@ -51,10 +54,13 @@ cd yggdrasil
 git checkout d696ee7a54bbf5775a88447bc40aae2259e8144c
 make
 ```
-
-
+Export `PREFIX` environment variable:
```

Comment:
you should update the section above on how to build only yggdrasil, e.g. `make bin` and define the `PREFIX` prior to it.

Reply:
done